### DISCUSSION TITLE
500 error

### DISCUSSION THREAD

**Discussion by Thisarahetz** [community/community#33134] [Other Feature Feedback, Questions, & Ideas]
I have been trying to login into my GitHub Account but it saying 500 error.
only that error come in main page(github.com) but I can go profile (github.com/username)
![image](https://user-images.githubusercontent.com/43540833/190322385-76ad185e-637e-4316-a39e-eed4462b6579.png)

**Comment by radar-blip** [comment]
Its happening to several people right now. Hopefully it will be fixed soon!
